### DISCUSSION TITLE
Default Password?

### DISCUSSION THREAD

**Discussion by DMac-eng** [josephdadams/TallyArbiter#708] [Q&A] [answered]
I downloaded TA to try and am trying to go to setup. I have searched the docs but I can't find what the username or password is to get to the setup page. BTW, is there any documentation/help other than the videos?

**Comment by hrueger** [comment, marked as answer]
Hi @DMac-eng ,
You can find the default credentials and more info here: https://josephdadams.github.io/TallyArbiter/docs/usage/control-interface

**Comment by DMac-eng** [comment]
Thanks.
